source/help/getting-started/configuring-notifications.md

```diff
@@ -20,11 +20,17 @@ Emails are sent for any mentions you receive if you have mattermost closed or ha
 - Configure the email address where notifications are sent in **Account Settings** > **General** > **Email** 
 
 ## Desktop Notifications
-These are browser notifications that appear in the corner of your main monitor. By default these notifications are sent for all activity while Mattermost is open. Desktop notifications are available on Firefox, Safari, and Chrome.
+These are browser notifications that appear in the corner of your main monitor. By default, these notifications are sent for all activity while Mattermost is open. Desktop notifications are available on Firefox, Safari, and Chrome.
 
-- Configure when desktop notifications are sent from **Account Settings** > **Notifications** > **Send Desktop Notifications**.
+- Configure when desktop notifications are sent from **Account Settings** > **Notifications** > **Desktop Notifications** > **Send Desktop Notifications**.
 - Configure channel specific desktop notifications from **Channel Menu** > **Notification Preferences** > **Send Desktop Notifications**. By default, all channels use the global setting configured in *Account Settings*.
 
+## Mobile Push Notifications
+If the Mattermost Android or iOS app is installed, push notifications can be sent to your mobile device. By default, these notifications are sent for all activity for any channel that is not being viewed on desktop.
```

Comment:
I think it's by design to send push notifications for mentions only (by default), so we just need to update it in the doc. 

Maybe confirm with @esethna as he owns notifications?